project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Blog heading is invisible
<img width="457" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/a8ea0359-446b-4162-bfe3-701f7cc2ef8f">